product.liquid

```diff
@@ -1,16 +1,25 @@
 <!-- EXPONEA start -->
 {% if product %}
 <script type="text/javascript">
+(function() {
+    var stock_level = 0;
+    {% for variant in product.variants %}
+    stock_level += {{ variant.inventory_quantity }};

```

Comment:
this makes the calculation (addition) to execute in Javascript, I think it's possible to write the template so it's summed in liquid already

Reply:
right, found a way how to do it and updated PR